capeinfra/datalake/datalake.py

```diff
@@ -431,6 +437,7 @@ def configure_sqs_lambda_target(self, jobs: list):
             environment={
                 "variables": {
                     "QUEUE_NAME": self.raw_data_queue.name,
+                    "DDB_REGION": self.aws_region,
```

Comment:
This is actually the wrong function for this environment variable. It should be passed to the s3 lambda trigger, not the SQS handler.

Reply:
lgtm